documentation/book/con-how-the-topic-operator-works.adoc

```diff
@@ -1,20 +1,26 @@
+// Module included in the following assemblies:
+//
+// topic-operator.adoc
+
+include::common/attributes.adoc[]
```

Comment:
Oh, we have to add this to each module? I didn't realise that, sorry.

Reply:
No worries, it's going to help us with findability down the road because we'll have a ton of modules.

Reply:
@ncbaratta I was referring to the `include::`. I still don't understand why it's worth having a list of all the places which include a given file when a simple `grep -r 'include::con-how-the-topic-operator-works.adoc' documentation/book/` can tell you that _and is always up to date_.

Reply:
For the include, yes, assume every file stands alone so that's why we need to include that there.

> Can you explain why these are necessary? Maintaining them is burdensome and it's info which you can figure out with a simple grep. So to me they add no value, but presumably you actually use them for something?

For this, we need these because these files are going to be added to a database that we're calling the module library down the road and having this info will help us understand where these pieces are coming from.  Also, while most engineers are working on the command-line, writers are working in the GUI and are using search functionality within our apps (like Atom) to find files and having these keywords in each file can be useful.